<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QWidget" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>554</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SABİT BİLGİLER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#main_form{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #3369fe, stop:1 #5834e9)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <widget class="QFrame" name="center_frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tab_constants">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::TabPosition::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::TabShape::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_personel_sabitleri">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="title">
          <string>Personeller</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="grp_turler">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Personel Türleri</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QListWidget" name="list_turler">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_turler">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">image:url(:/simgeler/ekle.ico)</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_turler">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grp_gorev">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Personel Görevleri</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QListWidget" name="list_gorev">
               <property name="styleSheet">
                <string notr="true">QListWidget {
    outline: 0; /* Seçim çerçevesini kaldırır */
    border: 1px solid #bcbcbc; /* Dış çerçeve */
}

QListWidget::item {
    border-bottom: 1px solid #d1d1d1; /* İŞTE BU: Satırları bölen belirgin çizgi */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_gorev">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_gorev">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grp_banka">
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Bankalar</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QListWidget" name="list_banka">
               <property name="styleSheet">
                <string notr="true">QListWidget {
    outline: 0; /* Seçim çerçevesini kaldırır */
    border: 1px solid #bcbcbc; /* Dış çerçeve */
}

QListWidget::item {
    border-bottom: 1px solid #d1d1d1; /* İŞTE BU: Satırları bölen belirgin çizgi */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_banka">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_banka">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grp_kan">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Kan Grupları</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QListWidget" name="list_kan">
               <property name="styleSheet">
                <string notr="true">QListWidget {
    outline: 0; /* Seçim çerçevesini kaldırır */
    border: 1px solid #bcbcbc; /* Dış çerçeve */
}

QListWidget::item {
    border-bottom: 1px solid #d1d1d1; /* İŞTE BU: Satırları bölen belirgin çizgi */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_kan">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_kan">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_adres_sabitleri">
         <attribute name="title">
          <string>İl-İlçeler</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QGroupBox" name="grp_iller">
            <property name="title">
             <string>İLLER</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="list_iller">
               <property name="styleSheet">
                <string notr="true">
QListWidget {
    outline: 0; /* Seçim çerçevesini kaldırır */
    border: 1px solid #bcbcbc; /* Dış çerçeve */
}

QListWidget::item {
    border-bottom: 1px solid #d1d1d1; /* İŞTE BU: Satırları bölen belirgin çizgi */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_iller">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_iller">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QGroupBox" name="grp_ilceler">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>İLÇELER</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="list_ilceler">
               <property name="styleSheet">
                <string notr="true">QListWidget {
    outline: 0; /* Seçim çerçevesini kaldırır */
    border: 1px solid #bcbcbc; /* Dış çerçeve */
}

QListWidget::item {
    border-bottom: 1px solid #d1d1d1; /* İŞTE BU: Satırları bölen belirgin çizgi */
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_ekle_ilceler">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/ekle.ico</normaloff>:/simgeler/ekle.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_sil_ilceler">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../icons/resource.qrc">
                   <normaloff>:/simgeler/remove.ico</normaloff>:/simgeler/remove.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_araclar">
         <attribute name="title">
          <string>Araçlar</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_vizeler">
         <attribute name="title">
          <string>Vizeler</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_surucu">
         <attribute name="title">
          <string>Sürücü Belgeleri</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resource.qrc"/>
 </resources>
 <connections/>
</ui>
